python/ray/WebUI.ipynb

```diff
@@ -4,741 +4,118 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Ray UI.\n",
+    "# Ray UI\n",
     "\n",
-    "#### To run the UI: \n",
-    "1. Kernel -> Restart and Run All \n",
-    "\n",
-    "\n",
-    "#### To hide the code:\n",
-    "\n",
-    "1. Install the hide_code extension.\n",
-    "```\n",
-    "pip install hide_code\n",
-    "jupyter nbextension install --py hide_code\n",
-    "jupyter nbextension enable --py hide_code\n",
-    "jupyter serverextension enable --py hide_code\n",
-    "```\n",
-    "2. Go to View -> Cell Toolbar -> Hide Code. \n",
-    "3. Press ESC then T"
+    "Start the UI with **Cell -> Run Cells**."
```

Comment:
This instruction doesn't work for me. I think it should be **Cell -> Run All**. Though it may be safer to just stick with **Kernel -> Restart and Run All**.